File: Lab1/Lab1Sharp/Lab1Sharp/FileManager.cs
namespace Lab1Sharp;

internal class FileManager
{
    public void GenerateFile(string fileName, string[] text) //створення і заповнення файлу
    {
        StreamWriter streamWriter = new StreamWriter(fileName, append: false);
        foreach (string line in text)
        {
            streamWriter.WriteLine(line);
        }

        streamWriter.Close();
    }

    public string ReadAllFromFile(string fileName) // зчитування тексту з першого файлу
    {
        StreamReader streamReader = new StreamReader(fileName);
        string text = streamReader.ReadToEnd();
        streamReader.Close();
        return text;
    }

    public void PrintFromFile(string fileName) // виведення тексту з файлу
    {
        StreamReader streamReader = new StreamReader(fileName);
        Console.WriteLine($"File name is {fileName}");
        Console.WriteLine(streamReader.ReadToEnd());
        streamReader.Close();
    }
}

File: Lab1/Lab1Python/file_manager.py
def generate_file(file_name, text: list[str]):  # створення і заповнення файлу
    file = open(file_name, "w")
    text = '\n'.join(text)
    file.write(text)
    file.close()


def read_all_from_file(file_name) -> str:  # зчитування тексту з першого файлу
    file = open(file_name)
    result = '\n'.join(file.readlines())
    file.close()
    return result


def print_from_file(file_name):  # виведення тексту з файлу
    file = open(file_name)
    print(f"\nFile name is {file_name}")
    print(''.join(file.readlines()))
    file.close()
